nntrainer/tensor/tensor.cpp

```diff
@@ -738,6 +740,14 @@ Tensor Tensor::sum(const std::vector<unsigned int> &axes, float alpha) const {
   return sum(axes, ret, alpha);
 }
 
+void Tensor::merge_axis(unsigned int axis1, unsigned int axis2) {
+  if (axis2 != axis1 + 1)
+    throw std::invalid_argument("Axis to be merged must be continuous.");
+
+  dim.setTensorDim(axis2, dim.getTensorDim(axis1) * dim.getTensorDim(axis2));
+  dim.setTensorDim(axis1, 1);
```

Comment:
Can we call Tensor::reshape instead?

Reply:
To use reshape, there has to be a switch making this slower in critical section depending on the location of axis.

Reply:
If we have change the reshape implementation, this will be likely to be untouched. If we have more detailed tests, I guess it will be fine :)

Reply:
Yes, will include tests for this with #1581.